src/mp4box/av01.rs

```diff
@@ -19,7 +19,7 @@ pub struct Av01Box {
     #[serde(with = "value_u32")]
     pub vertresolution: FixedPointU16,
     pub frame_count: u16,
-    pub depth: u16,
+    pub depth: u16, // I don't know what this is, but it is usually 24
```

Comment:
Pretty sure this is just bit depth, 3 channels * 8 bits per pixel = 24 bits total

Reply:
That's what I was about to write as well, especially since our av1 decoder also likes to talk in "total bits per full color sample" (i.e. 24/30/34). But then Emil told me on a call that he observed 24bit on a 10bit hdr video if I understood correctly.
Maybe worth pointing out here!

Reply:
I get `depth: 24, bit_depth: 10` when I print both